bsp_diff/common/kernel/linux-intel-lts2021/0030-Add-marvell-phy-suspend-resume-code.patch

```diff
@@ -0,0 +1,85 @@
+From 3f84f4ddf83036d5b839f2e859f63bd183861c9d Mon Sep 17 00:00:00 2001
+From: Zhao Ye <zhao.ye@intel.com>
+Date: Fri, 3 Nov 2023 10:58:47 +0800
+Subject: [PATCH] Add marvell phy suspend resume code
+
+Add marvell phy suspend resume code,
+Delete some useless warning.
+
+Tests:
+on Zeekr A1 board, ethernet can suspend/resume
```

Comment:
please not mention any customer name. if this is customer specific change, please use customer private repo.